api/desecapi/models.py

```diff
@@ -415,6 +419,32 @@ class Meta:
         managed = False
 
 
+# RR set types: the good, the bad, and the ugly
+# known, but unsupported types
+RR_SET_TYPES_UNSUPPORTED = {
+    'ALIAS',  # Requires signing at the frontend, hence unsupported in desec-stack
+    'DNAME',  # "do not combine with DNSSEC", https://doc.powerdns.com/authoritative/settings.html#dname-processing
+    'IPSECKEY',  # broken in pdns, https://github.com/PowerDNS/pdns/issues/9055 TODO enable support
+    'KEY',  # Application use restricted by RFC 3445, DNSSEC use replaced by DNSKEY and handled automatically
+    'OPENPGPKEY',  # large records not supported https://github.com/desec-io/desec-slave/issues/34 TODO enable support
+    'WKS',  # General usage not recommended, "SHOULD NOT" be used in SMTP (RFC 1123)
+}
+# restricted types are managed in use by the API, and cannot directly be modified by the API client
+RR_SET_TYPES_AUTOMATIC = {
+    # automatically managed by the backend:
+    'CDNSKEY', 'CDS', 'DNSKEY', 'KEY', 'NSEC', 'NSEC3', 'OPT', 'RRSIG',
```

Comment:
This changes the error message from "unsupported" to "automatically managed" record type. Admittedly, this never happens, but my idea was that if someone wanted to add a "KEY" to secure their records with some signatures, then we would tell them that we do that, instead of saying it's unsupported.

Reply:
ah, cool. I think it will be clear if the comment is `# corresponding functionality is automatically managed ...`